src/components/TileGallery/TileGallery.story.jsx

```diff
@@ -0,0 +1,241 @@
+        thumbnail={<Sunny32 fill="black" description="Icon" width={50} height={50} />}
+        icon={<CheckmarkFilled16 fill="#40be65" onClick={action('clicked')} />}
+        afterContent={overflowComponent}
+      />
+    </TileGallerySection>
+    <TileGallerySection title="More">
+      <TileGalleryItem
+        title="Card title"
+        description="card description"
+        moreInfoLink="https://www.ibm.com/br-pt/cloud/internet-of-things?mhsrc=ibmsearch_a&mhq=iot"
+        mode="grid"
+        thumbnail={<Sunny32 fill="black" description="Icon" width={50} height={50} />}
+        icon={<CheckmarkFilled16 fill="#40be65" onClick={action('clicked')} />}
+        afterContent={overflowComponent}
+      />
+    </TileGallerySection>
+  </Fragment>
+);
+
+export const galleryData = [
+  {
+    id: 'id1',
+    sectionTitle: 'Favorites',
+    galleryItems: [
+      {
+        title: 'Dashboard title',
+        description: 'More about your dashboard',
+        icon: <StarFilled16 />,
+        afterContent: overflowComponent,
+        thumbnail: <Activity32 />,
+        onClick: action('Card Clicked'),
+      },
+      {
+        title: 'Health',
+        description: 'More about your dashboard',
+        icon: <StarFilled16 />,
+        afterContent: overflowComponent,
+        thumbnail: <Sunny32 />,
+        onClick: action('Card Clicked'),
+      },
+      {
+        title: 'Activity',
+        description: 'More about your dashboard',
+        icon: <StarFilled16 />,
+        afterContent: overflowComponent,
+        thumbnail: <Activity32 />,
+        onClick: action('Card Clicked'),
+      },
+    ],
+  },
+  {
+    id: 'id2',
+    sectionTitle: 'Dashboard Category A',
+    galleryItems: [
+      {
+        title: 'Dashboard title',
+        description: 'More about your dashboard',
+        icon: <StarFilled16 />,
+        afterContent: overflowComponent,
+        thumbnail: <Activity32 />,
+        onClick: action('Card Clicked'),
+      },
+      {
+        title: 'Dashboard title',
+        description: 'More about your dashboard',
+        icon: <StarFilled16 />,
+        afterContent: overflowComponent,
+        thumbnail: <Sunny32 />,
+        onClick: action('Card Clicked'),
+      },
+      {
+        title: 'Dashboard title',
+        description: 'More about your dashboard',
+        icon: <StarFilled16 />,
+        afterContent: overflowComponent,
+        thumbnail: <Activity32 />,
+        onClick: action('Card Clicked'),
+      },
+    ],
+  },
+];
+
+storiesOf('Watson IoT Experimental|TileGallery', module)
+  .addDecorator(storyFn => <FullWidthWrapper>{storyFn()}</FullWidthWrapper>)
+
+  .add('Stateful TileGallery', () => (
+    <StatefulTileGallery
+      title="Dashboard"
+      hasSearch
+      hasSwitcher
+      hasButton
+      buttonText="Create +"
+      galleryData={galleryData}
+    />
+  ))
+  .add('basic example ', () => (
+    <div style={{ position: 'fixed', left: 0, right: 0, top: 0, bottom: 0 }}>
```

Comment:
I think we can wrap this with `<FullWidthWrapper>` to provide the same styling in a reusable way